CONTRIBUTING.md

```diff
@@ -37,6 +37,9 @@ Below is a short excerpt of some points in there:
 
     The old config generation system used impure shell scripts and could break in specific circumstances (see #1234).
 
+  When adding yourself as maintainer, in the same pull request make a separate commit with the message `maintainers: add <handle>`.
```

Comment:
```suggestion
  When adding yourself as maintainer, in the same pull request make a separate commit with the message `maintainers: add <handle>`.
Add the commit before those making changes to the package or module.
```

Reply:
Is it intentional, or is the indentation off?

Reply:
Ech, no, that was a typo.